source/manual/on-call.html.md

```diff
@@ -23,6 +23,22 @@ to keep GOV.UK running at night, on the weekends and on public holidays.
 - Logs are not as important as being available - if you need to lose some logs
   in order to bring the site back up, that's probably a good trade-off to make
 
+## On call checklist
+
+You should do these things before going on call so you're prepared.
+
+1. Have the numbers of other people on your shift saved in your phone. This
+includes whoever is on Escalations. Get these numbers from PagerDuty.
+2. Ensure you have an up to date local copy of the [Developer Docs][docs] repository
+and that you can build it.
+3. Make sure your [`fabric-scripts`][fabric] are up to date.
+4. Make sure you can VPN to the office or disaster recovery location
+5. Read these documents:
+    - [Stopping Nginx on the cache machines to fail to our static mirror](https://github.com/alphagov/fabric-scripts/blob/master/incident.py)
+    - [Emergency publishing](/manual/emergency-publishing.html)
+    - [Emergency publishing while origin is unavailable](/manual/fall-back-to-mirror.html)
+    - [Responding to an outage](https://gov-uk.atlassian.net/wiki/display/PLOPS/GOV.UK+Incidents)
```

Comment:
This page is a list of incidents, not instructions on responding to an outage. Is there a more appropriate page we could link to?

Reply:
Good call, fixed.

Reply:
:+1: (of course, the next question is, why is that page in Confluence rather than in the docs here?)